src/generic/RationalFunctionField.jl

```diff
@@ -22,10 +22,10 @@ base_ring(a::Rat) = base_ring(parent(a))
 
 parent(a::Rat) = a.parent
 
-data(x::Rat{T}) where T <: FieldElement = x.d::Frac{dense_poly_type(T)}
+data(x::Rat{T}) where T <: FieldElement = x.d::Union{Frac{dense_poly_type(T)}, Frac{mpoly_type(T)}}
```

Comment:
Is this the line you are talking about?
I don't like this line. Probably an extra parameter to the type is in order.